bin/rails

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env ruby
-APP_PATH = File.expand_path('../config/application', __dir__)
+APP_PATH = File.expand_path("../config/application", __dir__)
```

Comment:
The single- to double-quote changes in these boilerplate files are the result of running `rails app:update`. A follow-up ticket will enforce double-quotes for the stock files in `searchgov-style`, so that Rubocop and Rails are not enforcing conflicting styles.

Reply:
SRCH-3424 filed, but marked as a no-go as we already skip these files in rubocop.  Adding the comment here as well to close the loop.

Reply:
👍